text/0118-search-api.md

```diff
@@ -374,6 +380,76 @@ is equivalent to:
 }
 ```
 
+###### 3.1.2.1.13 Is Empty
```

Comment:
The section should be moved after `3.1.2.1.12 Geo Search` and before `3.1.2.2. Array Syntax`

Reply:
Yup, I did a mistake here 🙊